### PR TITLE
add pacmod3 depend for clang_tidy_pr

### PR DESCRIPTION
**Pull request by k0suke-murakami**
Signed-off-by: kosuke murakami <kosuke.murakami@tier4.jp>

## Description

fix clang_tidy_pr
https://github.com/tier4/AutowareArchitectureProposal.iv/runs/4259999021?check_suite_focus=true

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: .github/workflows/build_and_test_pr.yml [hunk 1]
```diff
@@ -101,8 +101,7 @@ jobs:
       - name: Install pip and Clang-Tidy
```
Comment (wep21) [inline .github/workflows/build_and_test_pr.yml:101]: @k0suke-murakami Could you add apt-transport-https in name?

Comment (k0suke-murakami) [reply .github/workflows/build_and_test_pr.yml]: Thanks, here is the commit https://github.com/tier4/AutowareArchitectureProposal.iv/pull/713/commits/10fccf7cfcd77de61197e63db7659aeb94ff4741

**Review by wep21 — APPROVED** [review]
LGTM

File: .github/workflows/build_and_test_pr.yml [hunk 2]
```diff
@@ -122,6 +121,11 @@ jobs:
           git config --local --unset-all http.https://github.com/.extraheader || true
           git config --global url.https://${{ secrets.REPO_TOKEN }}@github.com.insteadOf 'https://github.com'
 
+      - name: Add Autonomoustuff repository
```
Comment (kmiya) [inline .github/workflows/build_and_test_pr.yml:124]: TODO: Fix Autonomoustuff to AutonomouStuff.

**Comment by kmiya** [comment]
FYI: @KeisukeShima Clang-Tidy in metrics-reporter may also need these changes (in the future?).

**Comment by kenji-miyake** [comment]
I'd like to put these into https://github.com/tier4/autoware-github-actions (like `setup-autoware-depends.yaml`) instead of setting in each workflow/job.
@KeisukeShima @kmiya How do you feel?

**Pull request merged**